src/components/import/list-page.tsx

```diff
@@ -0,0 +1,49 @@
+import { TextInput, TextInputProps } from "@patternfly/react-core";
+import classNames from "classnames";
+
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+
+type TextFilterProps = Omit<TextInputProps, 'type' | 'tabIndex'> & {
+label?: string;
+parentClassName?: string;
+};
+export const KEYBOARD_SHORTCUTS = Object.freeze({
+focusFilterInput: '/',
+blurFilterInput: 'Escape',
+focusNamespaceDropdown: 'n',
+});
+
+export const TextFilter: React.FC<TextFilterProps> = (props) => {
```

Comment:
Looks like you pulled this in.  I think we need to format this.

Reply:
We can use ListPageFilter instead, will remove this TextFilter reference.